src/components/Message.tsx

```
@@ -197,7 +200,13 @@ const Message: FunctionComponent<MessageProps> = memo(({
 			<div class="message-content">
 				{/* Show content if available */}
 				{content && (
-					<div dangerouslySetInnerHTML={{ __html: marked(content) }} />
+					<div className="prose prose-sm max-w-none dark:prose-invert">
+						{isClient && ReactMarkdown ? (
+							<ReactMarkdown>{content}</ReactMarkdown>
+						) : (
+							<div dangerouslySetInnerHTML={{ __html: content }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)